### PR TITLE
[CI:DOCS] Fix up examples descriptions

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago @TomSweeneyRedHat @mheon More example cleanups for your enjoyment.

File: docs/source/markdown/podman-build.1.md.in [hunk 1]
```diff
@@ -480,59 +480,122 @@ using the architecture variant of the build host.
...
+```
 $ podman build --no-cache -t imageName .
+```
 
+Build image using layers and then remove intermediary containers even if the build fails.
+```
 $ podman build --layers --force-rm -t imageName .
+```
 
+Build image without using cache and do not remove intermediary containers even if the build succeeds:
```
Comment (edsantiago) [inline docs/source/markdown/podman-build.1.md.in:573]: intermediate, not intermediary

File: docs/source/markdown/podman-build.1.md.in [hunk 2]
```diff
@@ -480,59 +480,122 @@ using the architecture variant of the build host.
...
+Build image using the specified architectures and link to a single manifest on successful completion:
 ```
 $ podman build --arch arm --manifest myimage /tmp/mysrc
-
 $ podman build --arch amd64 --manifest myimage /tmp/mysrc
-
 $ podman build --arch s390x --manifest myimage /tmp/mysrc
+```
 
+Build image using multiple specified architectures and link to single manifest on successful completion:
```
Comment (edsantiago) [inline docs/source/markdown/podman-build.1.md.in:592]: As a naïve user, I would much rather see this as "same as above, but with just one command line"

File: docs/source/markdown/podman-build.1.md.in [hunk 3]
```diff
@@ -480,59 +480,122 @@ using the architecture variant of the build host.
...
+```
 $ podman build --platform linux/s390x,linux/ppc64le,linux/amd64 --manifest myimage /tmp/mysrc
+```
 
+Build image using multiple specified architectures and link to single manifest on successful completion:
```
Comment (edsantiago) [inline docs/source/markdown/podman-build.1.md.in:597]: Likewise, "same as above, just a different way"

**Review by edsantiago — COMMENTED** [review]
I've reviewed `build`. A tiny, tiny percent.

It will take me hours or days to review something this huge. I really appreciate your zeal to fix these, but in future could you please break these down into bite-size PRs?

I don't know how my day will go yet, and don't know if/when I'll be able to move on to `podman-commit`.

**Comment by rhatdan** [comment]
Broke this into 30 PRs to make it easier to review.

**Comment by TomSweeneyRedHat** [comment]
TY, TY for the split!

**Pull request closed without merging**